modules/transformations/redshift/test/integration/test_ST_DESTINATION.py

```diff
@@ -0,0 +1,43 @@
+from test_utils import run_query
+
+
+def test_destination_success():
+    results = run_query(
+        """SELECT ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(0, 0), 10, 90, 'kilometers')),
+        ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(-3.70325, 40.4167), 5, 45, 'kilometers')),
+        ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(-43.7625, -20), 150, -20, 'miles'))"""
+    )
+
+    assert str(results[0][0]) == 'POINT(0.089932 0)'
+    assert str(results[0][1]) == 'POINT(-3.661468 40.448488)'
+    assert str(results[0][2]) == 'POINT(-44.542881 -17.958279)'
+
+
+def test_destination_none():
+    results = run_query(
+        """SELECT ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            NULL, 10, 90, 'kilometers')),
+        ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(-3.70325, 40.4167), NULL, 45, 'kilometers')),
+        ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(-43.7625, -20), 150, NULL, 'miles')),
+        ST_ASTEXT(@@RS_PREFIX@@transformations.ST_DESTINATION(
+            ST_MakePoint(-43.7625, -20), 150, -20, NULL))"""
+    )
+
+    assert results[0][0] is None
+    assert results[0][1] is None
+    assert results[0][2] is None
+    assert results[0][3] is None
+
+
+def test_destination_default():
```

Comment:
The idea of the default test would be to compare the different overloaded versions of the function to check that the results are the same. Also if we are using an external library it could help to detect if they change the default params (steps, units...). So when you add the other version you can also update this test.